Plugins/SQL/Targets/PostgreSQL.cpp

```diff
@@ -36,13 +36,17 @@ void PostgreSQL::Connect(NWNXLib::ViewPtr<NWNXLib::Services::ConfigProxy> config
 
     const std::string port = "port=" + config->Get<std::string>("PORT", "5432");
 
-    // Build the m_connection string
-    std::string m_connectstring = host + " " + port + " " + db + " " + user + " " + pass;
+    // Build the m_connection string - this is used later in the PQping.
+    m_connectString = host + " " + port + " " + db + " " + user ;
 
-    m_log->Info("Connect String:  %s\n", m_connectstring.c_str());
+    // hide the password in the log file
+    m_log->Info("Connect String:  %s password=xxxxxxxx\n", m_connectString.c_str());
```

Comment:
You shouldn't need the \n here, since the logger writes \n each time.

Reply:
I will correct this.  I haven't tested the null parameter handling yet -- there may be some further fixes today.